nise/generators/ocp/ocp_generator.py

```diff
@@ -323,8 +367,42 @@ def _gen_pods(self, namespaces):
                         "mem_request_gig": mem_request_gig,
                         "mem_limit_gig": mem_limit_gig,
                         "pod_labels": self._gen_openshift_labels(),
+
+
+                    }
+
+                    ros_ocp_data_pods[pod] = {
+                        "namespace": namespace,
+                        "node": node.get("name"),
+                        "resource_id": node.get("resource_id"),
+                        "pod": pod,
+                        "deployment_name": self.fake.word() + "_" + self.fake.word(),
+                        "image_name": self.fake.word() + "-" + self.fake.word() ,
+                        "container_name":pod,
+                        "cpu_request_avg_container": randint(1,100),
+                        "cpu_request_sum_container": randint(1,100),
+                        "cpu_limit_avg_container": randint(1,100),
+                        "cpu_limit_sum_container": randint(1,100),
+                        "cpu_usage_avg_container": randint(1,100),
+                        "cpu_usage_min_container": randint(1,100),
+                        "cpu_usage_max_container": randint(1,100),
+                        "cpu_throttle_avg_container": randint(1,100),
+                        "cpu_throttle_max_container": randint(1, 100),
+                        "cpu_throttle_sum_container": randint(1, 100),
+                        "mem_request_avg_container": randint(1,100),
+                        "mem_request_sum_container": randint(1,100),
+                        "mem_limit_avg_container": randint(1,100),
+                        "mem_limit_sum_container": randint(1,100),
+                        "mem_usage_avg_container": randint(1,100),
+                        "mem_usage_min_container": randint(1,100),
+                        "mem_usage_max_container": randint(1,100),
+                        "mem-rss_usage_avg_container": randint(1,100),
+                        "mem-rss_usage_min_container": randint(1,100),
+                        "mem-rss_usage_max_container": randint(1,100),
```

Comment:
```suggestion
                        "mem_rss_usage_avg_container": randint(1,100),
                        "mem_rss_usage_min_container": randint(1,100),
                        "mem_rss_usage_max_container": randint(1,100),
```
These names need to use underscores.